Rakefile

```diff
@@ -4,12 +4,19 @@ require "rubygems"
 require 'rspec/core'
 require "rspec/core/rake_task"
 
+task :default => :spec
+
 RSpec::Core::RakeTask.new do |t|
   t.pattern = "spec/**/*_spec.rb"
   t.ruby_opts = "-w"
 end
 
-task :default => :spec
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop) do |t|
+  t.patterns = ['{lib,spec}/**/*.rb', 'Gemfile*'] +
+               %w[rack-test.gemspec Rakefile Thorfile]
+  t.options = %w[--display-cop-names --extra-details --display-style-guide]
```

Comment:
I would suggest setting these up in the `.rubocop.yml` file instead. That way, you will get the exact same behavior if you run `bundle exec rake rubocop` or `bundle exec rubocop` which is good.